cdk.json

```diff
@@ -28,6 +28,7 @@
       "aws",
       "aws-us-gov"
     ],
+    "@aws-cdk/core:checkSecretUsage": true,
```

Comment:
Nice, it never hurts to have those extra safety checks!

Reply:
It'll help in AT-7076 to make sure we're passing IAM User Access keys around safely too! So practical and timely!